build.assets/Dockerfile

```diff
@@ -1,18 +1,10 @@
-# This Dockerfile makes the "build box": the container used to build official
-# releases of Teleport and its documentation.
-#
-# Check the README to learn how to safely introduce changes to Dockerfiles.
-
-# Use Ubuntu 18.04 as base to get an older glibc version.
-# Using a newer base image will build against a newer glibc, which creates a
-# runtime requirement for the host to have newer glibc too. For example,
-# teleport built on any newer Ubuntu version will not run on Centos 7 because
-# of this.
+# This Dockerfile makes the "build box": the container used to
+# build Teleport Connect.
```

Comment:
This makes it sound like this image is used only for that, but I don't think that it's true. `build.assets/Makefile` includes quite a few targets that refer to `$(BUILDBOX)` but are not related to Connect.

Reply:
>This makes it sound like this image is used only for that

That was according to my best knowledge yesterday. Today, I checked a few places and update the comment :sweat_smile: